src/tester.rs

```diff
@@ -130,17 +130,8 @@ impl LangTester {
         self
     }
 
-    /// If set, defines what top-level lines will be treated as comments if they begin with
-    /// `comment_prefix`. Note that non-top-level lines are not checked against this.
-    ///
-    /// For example, if we set `comment_prefix` to `#` then:
-    ///
-    /// ```text
-    /// # Considered a top-level comment by lang_tester
-    /// Compiler:
-    ///   stdout:
-    ///     # A literal string beginning with `#`
-    /// ```
+    /// If set, defines what lines will be treated as comments if, ignoring the current level of
+    /// indentation, they begin with `comment_prefix`.
```

Comment:
This all LGTM. I wonder if here we should say:

> If set, defines what lines of a lang_tester test will be treated as comments if, ignoring the current level of indentation, they begin with `comment_prefix`.
>
> Note that this is not to be confused with another language's comment lines being in a test file when extracting the lang_tester test (e.g. using [LangTester::take_while].

Second sentence is iffy, but I hope you can see what I'm aiming at?

Reply:
Indeed. It's difficult to put into words.

Maybe we should just move on and wait for a user to squeal?

Reply:
> Maybe we should just move on and wait for a user to squeal?

I tend to agree.